ocs_ci/ocs/ocp.py

```diff
@@ -874,3 +875,104 @@ def verify_images_upgraded(old_images, object_data):
         f"All the images: {current_images} were successfully upgraded in: "
         f"{name}!"
     )
+
+
+def check_upgrade_completed(target_version):
```

Comment:
I think this name is confusing.
This function has nothing to do with an upgrade but just to confirm if the cluster is on the target version.
Could you please consider changing to something like: confirm_cluster_version